caps/capabilities.go

```diff
@@ -44,6 +44,10 @@ type Capability struct {
 	// capability, and also which information should be exchanged by these
 	// parties.
 	Type Type
+	// Attributes define additional key-value pairs that provide additional
+	// domain-specific information. They are closely related to the type of the
+	// capability.
+	Attributes map[string]string
```

Comment:
The Attrs abbreviation is very common, so we may as well use it here.

Also, the documentation may be shortened a bit without losing content:

    // Attrs are key-value pairs that provide type-specific capability details.